pkg/api/v1/schema.go

```diff
@@ -161,6 +167,9 @@ const (
 
 type CDRomTarget struct {
 	// Bus indicates the type of disk device to emulate.
+	// supported values: virtio, sata, scsi, ide
+	// see libvirt schema here:
+	// https://github.com/libvirt/libvirt/blob/v3.7-maint/docs/schemas/domaincommon.rng#L1693
```

Comment:
I thin the schema is really not helpful.

Reply:
fair enough, I will remove the references.